framework/src/Volo.Abp.AspNetCore.Mvc/Microsoft/AspNetCore/InProcess/CurrentDirectoryHelpers.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
```

Comment:
Move this to the Volo.Abp.AspNetCore package since it is independent from the MVC.
Also mart the class as static.